rgw/v2/tests/s3_swift/configs/test_cloud_transition.yaml

```diff
@@ -0,0 +1,18 @@
+# script: test_cloud_transitions.py
+config:
+  bucket_count: 1
+  objects_count: 10
+  objects_size_range:
+    min: 5
+    max: 15
+  test_ops:
+    create_bucket: true
+    create_object: true
+    endpoint: http://10.0.208.159:80
```

Comment:
Can we make this dynamic value

Reply:
@udaysk23 , hardcoded endpoint is just a placeholder. It will be passed through cephci in test runs.